keg_auth/model/__init__.py

```diff
@@ -92,6 +110,7 @@ def display_value(self):
     @classmethod
     def testing_create(cls, **kwargs):
         kwargs['password'] = kwargs.get('password') or randchars()
+        kwargs.setdefault('disable_date', None)
```

Comment:
Is this needed? The field is nullable.

Reply:
This was necessary because the disabled date was being set to current time and causing tests that expect the user to be enabled to fail. The Keg Elements MethodsMixin.testing_create skips fields with a default or server default, but since the default and server default are None, it skips the disable date.

I solved this by setting the server_default to sa.null().